pages/onboarding_project.ipynb

```diff
@@ -80,146 +130,413 @@
-      "\u001b[36mINFO\u001b[0m[0000] writing rbac definition to /home/anand/Documents/4n4nd/hitchhikers-guide/pages/apps/cluster-scope/components/project-admin-rolebindings/demo-team \n",
-      "\u001b[36mINFO\u001b[0m[0000] writing namespace definition to /home/anand/Documents/4n4nd/hitchhikers-guide/pages/apps/cluster-scope/base/core/namespaces/demo-project \n"
+      "/tmp/tmpjmgbmtb6/cluster-scope/overlays/prod/moc/smaug\n"
      ]
     }
    ],
    "source": [
-    "!opfcli create-project {NAMESPACE_NAME} {TEAM_NAME} -d \"{NAMESPACE_DISPLAY_NAME}\""
+    "%cd {WORKDIR}/cluster-scope/overlays/prod/{TARGET_CLUSTER}/\n",
+    "!kustomize edit add resource ../../../../base/core/namespaces/{NAMESPACE_NAME}"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 4. Adding namespace resources to the target cluster\n",
-    "\n",
-    "The following cell will add `../../../../base/core/namespaces/$NAMESPACE_NAME` as a resource to the kusomization for your target cluster (`cluster-scope/overlays/prod/$TARGET_CLUSTER/kustomization.yaml`).\n",
+    "The line added above by `kustomize edit` does not add the entry alphabetically, so we are going to sort it here ourselves (for human readability), you can also do it manually."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "kustomization_path = WORKDIR + \"/cluster-scope/overlays/prod/\" + TARGET_CLUSTER + \"/kustomization.yaml\"\n",
+    "with open(kustomization_path, \"r\") as f:\n",
+    "    kustomization = yaml.safe_load(f)\n",
+    "    kustomization['resources'].sort()\n",
+    "with open(kustomization_path, 'w') as f:\n",
+    "    yaml.dump(kustomization, f)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### 5. Add your quota to the namespace\n",
     "\n",
-    "\n"
+    "With the namespace manifest created, we now want to ensure the appropriate quota is added. If you picked the custom quota option you can skip to the next code block."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 59,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/tmp/tmpjmgbmtb6/cluster-scope/base/core/namespaces/demo-projecx\n"
+     ]
+    }
+   ],
+   "source": [
+    "%cd {WORKDIR}/cluster-scope/base/core/namespaces/{NAMESPACE_NAME}\n",
+    "if(not CUSTOM_QUOTA): \n",
+    "    !kustomize edit add component ../../../../components/resourcequotas/{QUOTA}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "If you require a custom quota, please enter the values for the resources you require:"
```

Comment:
Should we add a line that tells user to skip to step 6 if they don't need a custom quota?

Reply:
nah that's why I put an `if` clause, so it just won't count.